pillar/params.sls

```diff
@@ -67,6 +67,9 @@ ssl:
   crt_dir:        '/etc/pki'
   key_dir:        '/etc/pki'
 
+  velum_key: '/etc/pki/private/velum2.key'
+  velum_crt: '/etc/pki/velum2.crt'
```

Comment:
I guess these certificates are `bind`-mounted in the Velum container, so are you sure the container will get an updated version of the certificate?

Reply:
Sadly, you're right - the new certs did not get loaded.. I'll (probably) need to switch to a `docker restart`

Reply:
Try to mount in the container _the directory_ and not the files.

Reply:
I think this is more a result of the `USR2` signal on puma not reloading the certificates rather than the changes done by salt are not i visible on the container. Switching to a hard restart of the container should work, but it will bring a brief downtime.

Reply:
If we mount the directory, then velum has access to every certificate within it. This would be a security issue.. Besides, there is a second issue - Puma doesn't reload the certs from disk without a restart anyway.

Reply:
Then just `docker restart`! :-D